### ISSUE TITLE
Ability to extend participants video in external windows

### ISSUE THREAD

**Issue by roipoussiere** [jech/galene#70]
This way a user could manually place participants windows where he wants on its desktop, its more flexible.

This also allow a user to capture participants video independently with a screen capture tool (by targeting each participants window).

**Comment by MisterDA** [comment]
Modern browsers should support Picture-In-Picture mode with WebRTC videos. Do you see the button when hovering the video with the cursor?

**Comment by roipoussiere** [comment]
Yes and it works, but the problem with PiP mode is that the windows stays always on top, which doesn't fit my need.
